### DISCUSSION TITLE
When does vercel limits reset?

### DISCUSSION THREAD

**Discussion by Vette1123** [vercel/community#4084] [Help] [answered]
### Summary

Vercel limits are monthly reset right?, it should reset every month?, how can i know when it resets?, i can't find that in my profile.

### Steps to Reproduce

I want to know when my limits reset on vercel.

**Comment by z4nr34l** [comment]
Hello there,
yes, vercel limits are monthly - they are reseting every month's first day end.
The billing window is from month's 1st to next month's 1st.

**Comment by amyegan** [comment, marked as answer]
Hobby limits have a rolling 30 day window. When you look at the Usage tab, you'll see "Last 30 days" and a date range that is based on the current date.
<img width="583" alt="" src="https://github.com/vercel/community/assets/8247105/8d712ac4-7cd1-40f5-baf1-d9b89aeb135c">


Pro limits are reset monthly. When you look at the Usage tab on a Pro plan, you'll see "Current Billing Cycle" and a date range reflecting the start and end of the billing cycle.
<img width="589" alt="" src="https://github.com/vercel/community/assets/8247105/442bb936-e407-4436-ad4e-e47b0ac9e7ae">

**Reply by p1ckle-rick** [reply]
Hi @amyegan I need help understanding the "rolling 30 day window". I see the following in Vercel...

<img width="628" alt="Screenshot 2024-06-08 at 7 23 10 AM" src="https://github.com/vercel/community/assets/13596285/f39112b7-1ca7-495c-913a-2132a73e5c9e">

Does this mean all the **usage** before `May 8, 8pm` has been **erased/reset**?

**Reply by p1ckle-rick** [reply]
@amyegan any updates on this?

**Reply by amyegan** [reply]
Your Hobby team usage is always looking at the past 30 days, whatever the dates may be. Usage from 31 or more days back is considered previous, rather than current, usage.
